src/zope/errorview/http.py

```diff
@@ -40,6 +40,8 @@ def __init__(self, context, request):
 
     def update(self):
         self.request.response.setStatus(500)
+        self.request.response.setHeader(
+            'Content-Type', 'text/plain; charset=utf-8')
```

Comment:
And this.

Reply:
Idem.

Reply:
This has a spurious extra space.

Reply:
Which space is spurious?

Reply:
Seems curious that the request header in `browser` should *not* have a space after the semicolon, while the response header here does add that space.